web/packages/registry/src/polkadot_mainnet.registry.json

```
@@ -581,6 +588,14 @@
           "decimals": 18,
           "isSufficient": false
         },
+        "0xe9f6d9898f9269b519e1435e6ebaff766c7f46bf": {
+          "token": "0xe9f6d9898f9269b519e1435e6ebaff766c7f46bf",
+          "name": "vTAO",
+          "minimumBalance": "bigint:1",
+          "symbol": "vTAO",
+          "decimals": 18,
+          "isSufficient": false
+        },
```

If you use script `NODE_ENV=polkadot_mainnet npx ts-node build.ts` to generate the registry, the metadata actually looks like this:
```suggestion
        "0xe9f6d9898f9269b519e1435e6ebaff766c7f46bf": {
          "token": "0xe9f6d9898f9269b519e1435e6ebaff766c7f46bf",
          "name": "",
          "minimumBalance": "bigint:1",
          "symbol": "",
          "decimals": 0,
          "isSufficient": false
        },
```

That is because the metadata fetched here is from AssetHub. Do you have plans to open a ref on OpenGov to set the metadata for the token?